bundles/org.openhab.binding.lgthinq/src/main/java/org/openhab/binding/lgthinq/internal/errors/AccountLoginException.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.lgthinq.internal.errors;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * The {@link AccountLoginException}
+ *
+ * @author Nemer Daud - Initial contribution
+ */
+@NonNullByDefault
+public class AccountLoginException extends LGThinqException {
```

Comment:
Please add the serialVersionUID to all classes that imeplement Serializable

private static final long serialVersionUID = 1L;